modules/table/src/testFixtures/java/org/apache/ignite/internal/table/TxAbstractTest.java

```diff
@@ -2178,13 +2340,13 @@ protected void testTransactionAlreadyFinished(boolean commit, boolean checkLocks
         assertTrue(ex.getMessage().contains("Transaction is already finished."));
 
         ex = assertThrows(TransactionException.class, () -> accountsRv.delete(tx, makeKey(1)));
-        assertTrue(ex.getMessage().contains("Failed to enlist"));
+        assertTrue(ex.getMessage().contains("Transaction is already finished"));
```

Comment:
Should we use new utility method here that will also check code?

Reply:
yes, replaced this